db/seeds.rb

```diff
@@ -5,3 +5,9 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+skills = Skill.create([{ name: 'skill1', featured: true},
```

Comment:
Useless assignment to variable - `skills`.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.